benchmark/Makefile.am

```diff
@@ -16,9 +17,12 @@ endif
 if !HAVE_TIME
 $(error Did not find "time" in the PATH.)
 else
-TIMER = $(TIME) -v
+TIMER = $(TIME) -f "Real:%e\nSys:%S\nUser:%U\nCPU utilization:%P\nMaximum resident set size (kbytes):%M\nAverage resident set size (kbytes):%t\nAverage memory usage (kbytes):%K\nMajor page faults:%F\nMinor page faults:%R\nInvoluntarily context switches caused by memory:%W\nInvoluntarily context switches caused by time slice expirations:%c\nVoluntarily context switches:%w\nAverage unshared data area (kbytes):%D\nAverage unshared stack area (kbytes):%p\nAverage shared text space (kbytes):%X\nSystem's page size (bytes):%Z\nFilesystem inputs:%I\nFilesystem outputs:%O\nSocket messages received:%r\nSocket messages sent:%s\nSignals delivered to the process:%k"
```

Comment:
This will hurt portability (requiring a specific `time` implementation), but it's OK for this 1.0 release.